site/src/paradox/io/Type-Safe-BigQuery.md

```diff
@@ -55,7 +55,7 @@ import com.spotify.scio.bigquery.types.BigQueryType
 class Row
 
 // generate new query strings at runtime
-val newQuery = Row.query.format(args(0))
+val newQuery = Row.query(args(0))
```

Comment:
Why removing `format`? `query` is `String` and `String()` is `charAt`?

Reply:
If `fromQuery` of Row has multiple arguments, `Row` has a method `query` like below.
```
def query(queryArg$macro$1: String): String = "...".format(queryArg$macro$1)
```
This is generated here.
https://github.com/spotify/scio/blob/a6268507fb86be5fb9233231e3b4320eb351c03d/scio-bigquery/src/main/scala/com/spotify/scio/bigquery/types/TypeProvider.scala#L139-L144

If `fromQuery` of Row has a single argument, `Row.query(args(0))` will be compile error.  I think it is preferred because the query is not intended to be interpolated in such case.